File: BE/src/main/java/com/example/be/model/Bill.java
package com.example.be.model;

import javax.persistence.*;

@Entity
public class Bill {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String buyDate;
    private double totalPayment;
    private String nameUser;
    private String email;

    @ManyToOne
    @JoinColumn(name = "user_id", referencedColumnName = "id")
    private User user;

    @ManyToOne
    @JoinColumn(name = "product_id",referencedColumnName = "id")
    private Product product;

}

File: sprint2/src/app/model/bill.ts
import {User} from "./user";
import {Product} from "./product";

export interface Bill {
  id?:number;
  buyDate?:string;
  totalPayment?:number;
  nameUser?: string;
  email?: string;
  totalProduct?: string;
  user?: User;
  product?: Product;
}
